README_native.md

```diff
@@ -0,0 +1,88 @@
+# Python driver [![Build Status](https://travis-ci.org/bblfsh/python-driver.svg?branch=master)] ![Python version](https://img.shields.io/badge/python%20version-3.6-0.svg)
```

Comment:
Remove this file (`README_native.md`), most of it is redundant information. For some stuff such as notes for maintainers, we should create a separate DEVELOPERS.md or something like that. Let's discuss it for the sdk.